### DISCUSSION TITLE
Build Forward Node and Bandwidth

### DISCUSSION THREAD

**Discussion by alexkrycek** [Security-Onion-Solutions/securityonion#8609] [Unsupported Versions]
I'm planning on deploying a new distributed installation. We currently have several remote sites that connect over 10 Mbps SATCOM links. 

I can't seem to determine if SO has the ability to bandwidth throttle or batch schedule sensor data. Does anyone have insight on this? 

As far as building a Forward Node, the documentation doesn't go into details. Do I just install SO from the same ISO and run the soremote command?

Thank you!

**Comment by reyesj2** [comment]
You might consider building a heavy distributed deployment as opposed to standard forward node + search + manager. [Distributed deployment](https://docs.securityonion.net/en/2.3/architecture.html#distributed)

This will enable you to store logs locally as oppose to constantly saturating your satellite link.

Someone with more experience can chime in on benefits and/or drawbacks to this deployment type as it relates to your needs.

**Comment by dougburks** [comment]
We typically recommend against heavy nodes for most users, but you could try it and see if it works for your use case. If you try this, please pay attention to all of the notes and warnings at https://docs.securityonion.net/en/2.3/architecture.html#distributed.

> As far as building a Forward Node, the documentation doesn't go into details. Do I just install SO from the same ISO and run the soremote command?

Build nodes by running Setup, selecting the `DISTRIBUTED` install submenu, and choosing the `Existing Deployment` option. When prompted, join them to the manager node using the password that you created for the `soremote` account.

I've updated https://docs.securityonion.net/en/2.3/configuration.html#production-server-distributed-deployment to reflect this.
